### ISSUE TITLE
Console warning when no screenshots were taken

### ISSUE THREAD

**Issue by robfe** [percy/percy-webdriverio#15]
If none of my calls to `browser.percySnapshot` ever get called (which is conceivable if test's assertions are failing) the following error is printed to the console:

```
TypeError: Cannot read property 'then' of undefined
    at Object.wait (*\node_modules\fibers\future.js:449:15)
    at Object.percyFinalizeBuild (*\node_modules\wdio-sync\build\index.js:287:31)
    at Object.after (*\wdio.conf.js:236:17)
    at execHook (*\node_modules\wdio-sync\build\index.js:166:35)
    - - - - -
    at percyFinalizeBuild() - *\node_modules\wdio-sync\build\index.js:267:37
    at execute(<Function>) -     at *\node_modules\wdio-sync\build\index.js:316:3
```

This isn't blocking anything for me that i'm aware of, just wanted to raise it as a potential issue

**Comment by mfazekas** [comment]
@robfe thanks much for the report i've added a PR that raises and logs an error for finalizeBuild if no snapshots were issued.

**Comment by mfazekas** [comment]
This should be fixed by #19
